shared/chat/conversation/info-panel/header.js

```diff
@@ -38,26 +38,27 @@ type BigProps = {
 }
 
 const BigTeamHeader = ({channelname, teamname, onClick}: BigProps) => {
-  return [
-    <Text
-      key="bigTeamHeaderChannelName"
-      style={{alignSelf: 'center', marginTop: globalMargins.medium}}
-      type="BodyBig"
-    >
-      #{channelname}
-    </Text>,
-
-    <ClickableBox
-      key="bigTeamHeaderTeamName"
-      style={{...globalStyles.flexBoxRow, alignSelf: 'center', alignItems: 'center'}}
-      onClick={onClick}
-    >
-      <Avatar teamname={teamname} size={12} />
-      <Text type="BodySmallSemibold" style={{marginLeft: globalMargins.xtiny}}>
-        {teamname}
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'stretch'}}>
+      <Text
+        key="bigTeamHeaderChannelName"
```

Comment:
Do you still need this `key` and the one below?

Reply:
Oops, removed.